symfony/security-bundle/6.4/config/packages/security.yaml

```
@@ -7,7 +7,7 @@ security:
         users_in_memory: { memory: null }
     firewalls:
         dev:
-            pattern: ^/(_(profiler|wdt)|css|images|js)/
+            pattern: ^/(_profiler|_wdt|assets)/
```

```suggestion
            pattern: ^/_profiler|_wdt|assets|build/
```

There is no need for parenthesis if the union is the whole regex pattern.

Also, I think it's good to add `build` from Webpack Encore to the default pattern.